src/logic/map_objects/immovable_program.cc

```diff
@@ -396,49 +397,73 @@ void ImmovableProgram::ActRemove::execute(Game& game, Immovable& immovable) cons
 }
 
 /* RST
-
 seed
 ----
-Create a new immovable nearby with a chance depending on terrain affinity.
 
-Parameter syntax::
+.. function:: seed=\<immovable_name\> proximity:\<percent\>
+
+   :arg string \<immovable_name\>: The name of the immovable to create.
 
-  parameters ::= name factor
+   :arg percent proximity: The radius within which the immovable will seed is not limited and
+      is determined by repeatedly generating a random number and comparing it with the proximity
+      :ref:`map_object_programs_datatypes_percent` chance until the comparison fails. The higher
+      this number, the closer the new imovable will be seeded.
 
-Parameter semantics:
+   Finds a random location nearby and creates a new immovable with the given name there with a
+   chance depending on *this* immovable's terrain affinity. The chance that such a location will be
+   searched for in a higher radius is influenced by the ``proximity`` parameter. Note that this
+   program step will consider only *one* random location, and it will only seed there if the terrain
+   is well suited. This command may be used only for immovables with a terrain affinity. Example:
 
-``name``
-    The name of the immovable to create.
-``factor``
-    A natural integer in [1,254]. The radius within which the immovable will seed is not limited and
-    is determined by repeatedly generating a random number between 0 and 255 and comparing it with
-    ``factor`` until the comparison fails.
+.. code-block:: lua
 
-Finds a random location nearby and creates a new immovable with the given name there with a chance
-depending on *this* immovable's terrain affinity. The chance that such a location will be searched
-for in a higher radius can be influenced. Note that this program step will consider only *one*
-random location, and it will only seed there if the terrain is well suited. This command may be used
-only for immovables with a terrain affinity.
+     main = {
+         "animate=idle duration:20s",
+         "remove=chance:11.72%",
+         -- Select a location with a chance of 19.53% in the base radius,
+         -- then expand the radius and try again with a chance of 19.53%.
+         -- Repeat until a location has been selected, then plant an
+         -- 'umbrella_red_wasteland_sapling' if the terrain affinity check
+         -- for this immovable succeeds.
+         "seed=umbrella_red_wasteland_sapling proximity:19.53%",
+         "animate=idle duration:20s",
+         "remove=chance:7.81%",
+         "grow=umbrella_red_wasteland_old",
```

Comment:
the lines after the seed command seem to be a copy paste error, as they do not result in a meaningful immovableprogram

Reply:
This is 100% what's in the actual program. There's a 92.19% chance that the remove program fails, in which case the tree might grow.